src/openclsim/critical_path/dependencies_from_model.py

```diff
@@ -0,0 +1,243 @@
+"""
+module contains DependenciesFromModel that inherits from critical_path.base_cp.BaseCP
+
+in original commit this module would contain may lines and many (hidden/protected)
+helper functions and methods which ar enot all listed here
+"""
+import networkx as nx
+
+from openclsim.critical_path.base_cp import BaseCP
+
+
+class DependenciesFromModel(BaseCP):
+    """build dependecies from recorded_activities_df"""
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def get_dependency_list(self):
+        """
+        get dependencies from simulation setup and
+        cross reference with recorded activities to get dependencies of relevant instances
+        requires self.object_list, self.activity_list and self.recorded_activities_df
+
+        Notes
+        ---------
+        this method is likely to provide too few dependencies and might raise warnings
+        for unsupported start event conditions, activities et cetera
+
+        this method contains approx 1000 (!) lines in original commit
```

Comment:
First word of the scentence starts with a capital.

no need to descript the number of lines of code in the docstring

Reply:
ok